tests/module_1/test_meteo_api.py

```diff
@@ -1,7 +1,53 @@
-""" This is a dummy example to show how to import code from src/ for testing"""
+import pandas as pd
```

Comment:
I suggest you to have a look at pytest packag, the use of fixtures for data, and the Mock object of unittest package